CollapseLauncher/Classes/Properties/InnerLauncherConfig.cs

```diff
@@ -89,14 +92,13 @@ public static string GetComboBoxGameRegionValue(object obj)
         public static List<StackPanel> BuildGameTitleListUI()
         {
             List<StackPanel> list = new List<StackPanel>();
-            foreach (string title in ConfigV2GameCategory)
+            foreach (string title in LauncherMetadataHelper.GetGameNameCollection())
```

Comment:
Suppress warning because metadata will never be empty, or check if value is `null`.